src/videojs-contrib-hls.js

```diff
@@ -208,11 +208,19 @@ class HlsHandler extends Component {
         Object.defineProperty(_player, 'hls', {
           get: () => {
             videojs.log.warn('player.hls is deprecated. Use player.tech_.hls instead.');
+            // fired the uasge event when player.hls is accessed
+            tech.trigger({type: 'usage', name: 'hls-player-access'});
             return this;
           }
         });
       }
     }
+    // Object.defineProperty(tech, 'hls', {
```

Comment:
trying to do the detection of accessing `player.tech_.hls` here(fire `hls-tech-access` event when `player.tech_.hls` is accessed). was planning to define a get method for `tech_.hls` for the detection purpose but it seems that doesn't work. Do you have any good idea about how to detect the `hls-tech-access`?